cedar-policy-core/src/entities/json/schema_types.rs

```diff
@@ -166,7 +185,18 @@ impl std::fmt::Display for SchemaType {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
             Self::Bool => write!(f, "bool"),
-            Self::Long => write!(f, "long"),
+            Self::Long { min, max } =>
```

Comment:
I think you could do:
`Self::Long{LONG_MIN, LONG_MAX} => write!(f, "long with no bounds")`